src/interfaces/code.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 OpenStax Poland
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license text.
+
+import { Element } from 'slate'
+
+/** A code fragment */
+export interface Code {
```

Comment:
This should extend `Element` interface because currently `Editor.above(editor, { match: Code.isCode })` is returning wrong type: `NodeEntry<Ancestor> | undefined` instead of `NodeEntry<Code> | undefined`